ci-scripts/yocto.groovy

```diff
@@ -21,12 +21,6 @@ void startVagrant(boolean kill=true) {
             shutdownVagrant()
         }
 
-        // Pull Docker container from Docker Hub
-        sh "docker pull pelagicore/pelux_ubuntu1604"
-
-        // Tag container as 'pelux'
-        sh "docker tag pelagicore/pelux_ubuntu1604 pelux"
-
         sh "vagrant up"
```

Comment:
Why remove this?

Reply:
We've removed usage of the pre-built container to simplify debugging of the CI/CD issues. Once it's stable, we'll update the image on Docker Hub.

Reply:
Oh I didn't know this.

Reply:
@sashko Just put what you wrote here in the commit message. A perfect "why". :) The commit message as it is now only contains "what". :(